OloEditor/src/Panels/SceneHierarchyPanel.cpp

```
@@ -49,6 +49,105 @@ namespace OloEngine
                     m_SelectionContext = m_Context->CreateEntity("Empty Entity");
                 }
 
+                if (ImGui::BeginMenu("Create UI"))
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)